db/seeds/test_provider_populator.rb

```diff
@@ -13,27 +13,29 @@ class TestProviderPopulator
     "Ahern & Co." => [808, %w[9F808X:21 3X808Z:22]],
     "Groves & Co." => [989, %w[7F809X:23 4X809Z:24]],
     "Sugarman & daughters" => [810, %w[6F810X:25 6X810Z:26]],
+    "Test Connor Co." => [118, %w[5F809X:27 5X809Z:28]],
     "EDWARD HAYES LLP" => [19_537, %w[1T462U:85605]],
     "LAWRENCE & CO SOLICITORS CDS LLP" => [33_230, %w[0Q974B:85981]],
   }.freeze
 
   TEST_PROVIDERS = {
-    "test1" => ["Test & Co", "test1@example.com", 100, 370],
-    "will-c" => ["Will-c & Co.", "william.clarke@digital.justice.gov.uk", 101, 424],
-    "test-sch" => ["Test-sch & Co.", "apply-for-civil-legal-aid@digital.justice.gov.uk", 102, 587],
-    "test2" => ["Test2 & Co.", "test2@example.com", 103, 588],
-    "test3" => ["Test3 & Co.", "really-really-long-email-address@example.com", 104, 589],
-    "firm1-user1" => ["Firm1 & Co.", "firm1-user1@example.com", 105, 590],
-    "firm1-user2" => ["Firm1 & Co.", "firm1-user2@example.com", 106, 591],
-    "firm2-user1" => ["Firm2 & Co.", "firm2-user1@example.com", 107, 592],
-    "sr" => ["Richards & Co.", "stephen.richards@digital.justice.gov.uk", 108, 593],
-    "ahernk" => ["Ahern & Co.", "katharine.ahern@digital.justice.gov.uk", 109, 594],
-    "MARTIN.RONAN@DAVIDGRAY.CO.UK" => ["David Gray LLP", "martin.ronan@example.com", 494_000, 5027],
-    "BENREID" => ["Test firm for portal login", "benreid@example.co.uk", 107, 592],
+    "test1" => ["Test & Co", "test1@example.com", 100],
+    "will-c" => ["Will-c & Co.", "william.clarke@digital.justice.gov.uk", 101],
+    "test-sch" => ["Test-sch & Co.", "apply-for-civil-legal-aid@digital.justice.gov.uk", 102],
+    "test2" => ["Test2 & Co.", "test2@example.com", 103],
+    "test3" => ["Test3 & Co.", "really-really-long-email-address@example.com", 104],
+    "firm1-user1" => ["Firm1 & Co.", "firm1-user1@example.com", 105],
+    "firm1-user2" => ["Firm1 & Co.", "firm1-user2@example.com", 106],
+    "firm2-user1" => ["Firm2 & Co.", "firm2-user1@example.com", 107],
+    "sr" => ["Richards & Co.", "stephen.richards@digital.justice.gov.uk", 108],
+    "ahernk" => ["Ahern & Co.", "katharine.ahern@digital.justice.gov.uk", 109],
+    "MARTIN.RONAN@DAVIDGRAY.CO.UK" => ["David Gray LLP", "martin.ronan@example.com", 494_000],
+    "BENREID" => ["Test firm for portal login", "benreid@example.co.uk", 107],
     "HFITZSIMONS@EDWARDHAYES.CO.UK" => ["EDWARD HAYES LLP", "hfitzsimons@example.com", 2_453_773],
     "LHARRISON@TBILAW.CO.UK" => ["LAWRENCE & CO SOLICITORS CDS LLP", "LHARRISON@example.com", 954_474],
-    "CG" => ["Groves & Co.", "chris.groves@digital.justice.gov.uk", 110, 599],
-    "jsugarman" => ["Sugarman & daughters", "joel.sugarman@digital.justice.gov.uk", 111, 600],
+    "CG" => ["Groves & Co.", "chris.groves@digital.justice.gov.uk", 110],
+    "jsugarman" => ["Sugarman & daughters", "joel.sugarman@digital.justice.gov.uk", 111],
+    "cm" => ["Test Connor Co.", "connor.mcquillan@digital.justice.gov.uk", 110],
```

Comment:
on this, yes they do not look to be used, perhaps @colinbruce or @stephenrichards know why these are here. it looks safe to remove them.

Reply:
It looks like @willc-work removed the 4th value [here](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/1669/commits/ddceed3a65094e4f06ed18d273680a0387f18358#diff-d2079fee4b31059a89fd5977fdaed6a72e405a229aeb6b8fa3e713ae66fa4b36L56) when we switched authentication - good to remove :+1: